configs/components/ruby.rb

```diff
@@ -17,6 +17,11 @@
   pkg.replaces 'pe-ruby-ldap'
   pkg.replaces 'pe-rubygem-gem2rpm'
 
+  pkg.conflicts 'pe-puppet-enterprise-release', '2016.3.0.0'
```

Comment:
I'll double-check. My understanding was the conflict is less than the specified version.

https://github.com/puppetlabs/vanagon/blob/master/resources/rpm/project.spec.erb#L59-L61

https://github.com/puppetlabs/vanagon/blob/master/resources/deb/control.erb#L18